addons/sale_purchase_stock/tests/test_lead_time.py

```diff
@@ -61,3 +62,33 @@ def test_supplier_lead_time(self):
 
         po = self.env['purchase.order'].search([('partner_id', '=', self.vendor.id)])
         self.assertEqual(po.order_line.price_unit, seller.price)
+
+    @freeze_time('2024-03-01')
```

Comment:
Do you really need to freeze time for this test? Or can you just make the commitment date `fields.Date.today()`? It's better if we can avoid it since freeze time will also affect the log dates when the test is run.